esp/logging/logginglib/logthread.cpp

```diff
@@ -157,8 +157,7 @@ bool CLogThread::queueLog(IEspUpdateLogRequestWrap* logRequest)
 {
     unsigned startTime = (getEspLogLevel()>=LogNormal) ? msTick() : 0;
```

Comment:
I wonder if we're actually saving any process cycles, etc. by avoiding msTick() but calling getEspLogLevel